pkgs/development/compilers/yosys/default.nix

```diff
@@ -32,7 +32,7 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ tcl readline libffi python3 bison flex ];
+  buildInputs = [ gcc tcl readline libffi python3 bison flex ];
```

Comment:
Why does this need gcc?

Reply:
I guess you should ask the authors…

There is a “config-clang” target in the makefile, but I did not manage to get the build go through (I got an error related to C++ header files). Should I investigate further or is the work-around of using GCC acceptable?

Reply:
Sorry for the delay. Just adding gcc doesn't work properly so I don't think it's actually needed. But if you don't want to spend more time on this, overriding the stdenv in all-packages.nix is the proper way to use gcc in all platforms.

```nix
{
  foo = callPackage ./foo { stdenv = gccStdenv; };
}
```